src/jobs/eventRepetitions.js

```diff
@@ -1,63 +1,93 @@
+/* eslint-disable no-loop-func */
```

Comment:
Je ne crois pas que ce soit utile, si ?

Reply:
si, il m'aime pas mess filter dans la boucle for ...

Reply:
Ha ok trop bizarre je l'ai enlevé et le linter ne me disait rien ^^